src/Microsoft.OData.Core/UriParser/Binders/FunctionCallBinder.cs

```diff
@@ -106,67 +109,95 @@ internal static FunctionSignatureWithReturnType MatchSignatureToUriFunction(stri
                 // we specifically want to find just the first function that matches the number of arguments, we don't care about
                 // ambiguity here because we're already in an ambiguous case where we don't know what kind of types
                 // those arguments are.
-                signature = signatures.FirstOrDefault(candidateFunction => candidateFunction.ArgumentTypes.Count() == argumentCount);
-                if (signature == null)
+                var found = nameSignatures.FirstOrDefault(pair => pair.Value.ArgumentTypes.Count() == argumentCount);

```

Comment:
>var [](start = 16, length = 3)

Please replace var with type (KeyValuePair<string, FunctionSignatureWithReturnType>)

Reply:
Fixed other "var "in dozen other places in this file as well. But I heard that using "var " is encouraged these days since type inference was introduced as new C# feature. I am OK with either way, understanding that many developers (including good ones) favors strong type.
I don't think we need to cleanup all files related to this since we have time crunch. We might need team discussion when topic related to coding standard comes up.

---
In reply to: [190097866](https://github.com/OData/odata.net/pull/1156#discussion_r190097866) [](ancestors = 190097866)